lib/combinations.js

```diff
@@ -0,0 +1,16 @@
+function combinations(collection, size) {
+  const len = collection.length;
+
+  if (size > len) return [];
+  if (!size) return [[]];
+  if (size === len) return [collection];
```

Comment:
I would convert these from one liners to:
if (foo) { THING HERE }

Reply:
This is technically undefined in airbnb styleguide though I'm happy to have it up for discussion.

http://airbnb.io/javascript/#blocks

Reply:
Apologies, the styling ruined my point, I meant like this part:
```
// good
if (test) {
  return false;
}
```